pdc/apps/contact/models.py

```diff
@@ -8,11 +8,14 @@
 from django.db import models
 from django.db.models.query import QuerySet
 from django.forms.models import model_to_dict
+from django.core.exceptions import ValidationError
 
 
 class ContactRole(models.Model):
 
     name = models.CharField(max_length=128, unique=True)
+    count_limit = models.IntegerField(default=1)
```

Comment:
how about use `count_contact_limit`? for my first glimpse, I understood it as the count limit of Role :-)